src/index.js

```diff
@@ -45,17 +45,18 @@
  * The main interface is the {@link analyze} function.
  * @module escope
  */
+"use strict";
 
 /*jslint bitwise:true */
 
-import assert from 'assert';
+const assert = require('assert');
 
-import ScopeManager from './scope-manager';
-import Referencer from './referencer';
-import Reference from './reference';
-import Variable from './variable';
-import Scope from './scope';
-import { version } from '../package.json';
+const ScopeManager = require('./scope-manager').default;
+const Referencer = require('./referencer').default;
+const Reference = require('./reference').default;
+const Variable = require('./variable').default;
+const Scope = require('./scope').default;
```

Comment:
I was planning to do a style clean up PR at the end that would include this. Wanted to keep changes in each PR to a minimum but can change it now if people would prefer